src/money-formatter.js

```diff
@@ -2,11 +2,18 @@
 import currencyFormatData from 'currency-format';
 import { formatDigits, isNumeric } from './utils';
 
-const getCurrencyData = (code) =>
+const getCurrencyData = code =>
   currencyFormatData[code.toUpperCase()];
 
-export const formatSimple = (currencyName, amount, fractionSize) =>
-  `${formatDigits(amount, fractionSize)} ${currencyName}`;
+export const formatSimple = (currencyName, amount, fractionSize) => {
```

Comment:
Could you please update API docs in the README to make it clear that `currencyName` argument is optional?